### ISSUE TITLE
Translations in rc1 do not (fully) work

### ISSUE THREAD

**Issue by vanto** [torchbox/wagtail#1452]
When I'm bootstrapping a new project from the project template via `wagtail start website` and change LANGUAGE_CODE to "de-de", the translation is very half-baked, although the translations seem to be available in the .po files.

Here is what I did.

1. `pip install -U wagtail==1.0rc1`
2. `wagtail start website`
3. `cd website`
4. `pip install -r requirements.txt`
5. `sed -i '' 's/en-us/de-de/' website/settings/base.py` to switch the locale to German.
6. `./manage.py migrate && ./manage.py createsuperuser`
7. `./manage.py runserver 0.0.0.0:8000`

When I open the wagtail admin in a browser, the login screen is still english but with some German words. It become even stranger after logging in: Most of the admin page is in English but the Settings menu and the Search box are in German. Documents and Images seem to be completely in English, the Page explorer.

![image](https://cloud.githubusercontent.com/assets/11374/8407871/3a134c96-1e6b-11e5-8dcd-88e00017bc55.png)

![image](https://cloud.githubusercontent.com/assets/11374/8407894/56f201e0-1e6b-11e5-99be-62b09100ca14.png)

Am I missing anything? The only pattern I could recognize are the missing .mo files in some modules. Could that be the reason?

**Comment by kaedroho** [comment]
It appears some .mo files are missing: https://github.com/torchbox/wagtail/tree/master/wagtail/wagtailimages/locale/de/LC_MESSAGES

Most likely caused by invalid syntax in the files returned by transifex

**Comment by vanto** [comment]
I'm glad to help. Thanks for the confirmation. I just ran the compilemessages command and got the following errors. I guess your assumption is right and there is something wrong with some russian translations.

```
CommandError: Execution of msgfmt failed: /Users/vanto/dev/git/wagtail/wagtail/wagtailadmin/locale/ru/LC_MESSAGES/django.po:245: 'msgid' and 'msgstr[2]' entries do not both begin with '\n'
/Users/vanto/dev/git/wagtail/wagtail/wagtailadmin/locale/ru/LC_MESSAGES/django.po:351: 'msgid' and 'msgstr[2]' entries do not both begin with '\n'
msgfmt: found 2 fatal errors

CommandError: Execution of msgfmt failed: /Users/vanto/dev/git/wagtail/wagtail/wagtaildocs/locale/ru/LC_MESSAGES/django.po:162: 'msgid' and 'msgstr[2]' entries do not both begin with '\n'
msgfmt: found 1 fatal error

CommandError: Execution of msgfmt failed: /Users/vanto/dev/git/wagtail/wagtail/wagtailimages/locale/ru/LC_MESSAGES/django.po:158: 'msgid' and 'msgstr[2]' entries do not both begin with '\n'
msgfmt: found 1 fatal error

**Comment by gasman** [comment]
Fixed in #1455
